Cargo.toml

```diff
@@ -48,6 +48,9 @@ test_util = { path = "./test_util" }
 deno_lockfile = "0.17.1"
 deno_media_type = { version = "0.1.1", features = ["module_specifier"] }
 
+denokv_proto = "0.1.0"
+denokv_sqlite = "0.1.0"
```

Comment:
Curious why no underscore between deno and kv?

Reply:
Everything `deno_` prefixed is part of the `deno` binary. All the `denokv_` things are components of the `denokv` binary that just _happen_ to also be pulled into `deno`.